File: Beginner/Chef and Price Control (PRICECON)/price.py
t = int(input())

while t:
    N, K = map(int, input().split())

    P = list(map(int, input().split()))
    tot = sum(P)

    for i in range(len(P)):
        if P[i]>K:
            P[i] = K

    rev = sum(P)
    print(tot - rev)

    t = t-1

File: Beginner/Chef and Price Control (PRICECON)/price.cpp
#include <bits/stdc++.h>
using namespace std;
int main()  {
    int t;
    cin>>t;
    
    while(t--) {
        int N,K, sum=0, tot=0;
        cin>>N>>K;
        int arr[N];
        for(int i=0;i<N;i++) {
            cin>>arr[i];
            tot = tot+arr[i];
            if(arr[i]>K) 
                arr[i]=K;
            sum = sum+arr[i];
        }
        cout<<tot-sum<<endl;
    }
}
